docs/src/main/sphinx/connector/pushdown-char-varchar-case.fragment

```diff
@@ -0,0 +1,17 @@
+Predicate pushdown support
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector does not support pushdown of inequality predicates, such as
+``!=``, ``>``, or ``<``, on columns with :ref:`textual types <string-data-types>`
```

Comment:
This isn't foolproof. Someone may confuse it to mean this only applies to `!=`, `>` and `<` and not things like `BETWEEN`.

WDYT about adding `etc.` to the end of the list?

Reply:
Maybe replace `textual types` with `character string types`? (It's the formal term used in literature as well as documentation of other systems like MySQL and SQL Server)

Reply:
The Google dev docs style guide, which we follow, strongly discourages use of "etc" in docs. Preference would be that we keep the "such as" wording and just try to provide a diverse (if not complete) list of applicable terms. I included `BETWEEN` as an example if that helps.